pillar/dev/top.sls

```diff
@@ -8,6 +8,7 @@ base:
     - tls
     - users.*
     - postgres.clusters
+    - secrets.sentry
```

Comment:
@ewdurbin would we need this really since we dont expect it to run in dev and the "do we run this?" checks are if the keys in pillar secrets exist

Reply:
Yeah, no need to _always_ have it on. Maybe just comment this line out so people have an idea where to turn it on.